README.md

```diff
@@ -49,6 +49,13 @@ $ make
 $ make verify  # (optional)
 ```
 
+## CMake (Mac)
+
+```
+$ mkdir build && cd build
+$ cmake -D DARWIN:BOOL=ON .. -G "Xcode"    # Generate Xcode project.
```

Comment:
And simply `cmake -G Xcode` does not work? BTW which cmake version do you have?

Reply:
Yes, I tested it myself on the Mac. I modified the Cmakelist.txt file.

Reply:
cmake version 3.16.5